alerts/cluster-network-operator/OVNKubernetesControllerDisconnectedSouthboundDatabase.md

```diff
@@ -0,0 +1,53 @@
+# OVNKubernetesControllerDisconnectedSouthboundDatabase
+
+## Meaning
+
+The `OVNKubernetesControllerDisconnectedSouthboundDatabase` alert is triggered
+when the OVN controller is not connected to OVN southbound database
+for more than 5 minutes.
+
+## Impact
+
+Networking control plane is degraded on the node. Existing workloads
+on the node may continue to have connectivity but any networking configuration
+update will not be applied.
+
+## Diagnosis
+
```

Comment:
I think we should add "Fix alerts before continuing" for "parent" alert https://github.com/openshift/runbooks/blob/master/alerts/cluster-network-operator/NodeWithoutOVNKubeNodePodRunning.md like here https://github.com/openshift/runbooks/blob/master/alerts/cluster-network-operator/NoOvnMasterLeader.md#fix-alerts-before-continuing

Reply:
So if this alert fires, it means that ovnkube-node is running, then we can add a step to check if this alert only fires for 1 node, then continue with debugging this node, or if all nodes have this alert, then the problem should be with sbdb.
 NoOvnMasterLeader shouldn't be directly the reason, because sbdb may have leader and be running at the same, but https://github.com/openshift/runbooks/blob/master/alerts/cluster-network-operator/NoRunningOvnMaster.md I guess can be the reason (since it guarantees no sbdb members are running)

Reply:
I'll try to add a step to check for sbdb errors regardless of how many nodes are failing to connect since it might be helpful in both cases. 
To clarify what I meant above: I added the "Fix alerts before continuing" section for `NoRunningOvnMaster` just like in the link, I agree the is no point to include `NoOvnMasterLeader`.

Reply:
Ah right, sorry I got confused with these alert names